specs/src/specs/data_structures.md

```diff
@@ -797,6 +794,10 @@ Assigns validator's pending commission to a delegation.
 
 Adds delegation's pending rewards to voting power.
 
+### PayForBlobData
+
+// TODO: <https://github.com/celestiaorg/celestia-app/issues/1435>
```

Comment:
No, it's not strictly necessary. I'm going to merge as-is to avoid re-requesting reviews.

I expect the TODO will be resolved and the link will be removed prior to merging specs-staging into main.

Reply:
Will remove b/c needed to re-request reviews.